TEStribute/distance.py

```diff
@@ -0,0 +1,27 @@
+import geopy.distance
+from ip2geotools.databases.noncommercial import DbIpCity
+import socket
+
+
+def ReturnDistance(ip1, ip2):
```

Comment:
Since I mentioned "production code", I think it's a good idea to use Python's fancy new `typing` to give us some more security. Have a look at the docs and implement it for the functions you already have and then, going forward, do it immediately when you add a new function. It's very annoying to do this for a big set of functions at the very end, and it's good practice for writing stable code (especially if it is going to be used in web services).

Reply:
Also, please use doc strings for all your modules and functions. Again, it is really really annoying to do this at the very end for all your modules and functions, and it is good practice to have this be the very first thing you do after creating a module or defining a function signature. Have a look at some guidelines on how to write good doc strings. It is really important to have them, as docs and instructions for the usage of your tools and functions is auto-generated from them.